ERROR - 2025-01-02 09:47:32 --> Error connecting to the database: ErrorException: mysqli::real_connect(): Error while reading greeting packet. PID=15564 in D:\xampp\htdocs\debitnote\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'mysqli::real_co...', 'D:\\xampp\\htdocs...', 171)
#1 D:\xampp\htdocs\debitnote\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'debitnote', 3306, '', 0)
#2 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#3 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#4 D:\xampp\htdocs\debitnote\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#5 D:\xampp\htdocs\debitnote\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#6 D:\xampp\htdocs\debitnote\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#7 D:\xampp\htdocs\debitnote\app\Controllers\Login.php(26): CodeIgniter\BaseModel->findAll()
#8 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(902): App\Controllers\Login->index()
#9 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#10 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 D:\xampp\htdocs\debitnote\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}

Next mysqli_sql_exception: MySQL server has gone away in D:\xampp\htdocs\debitnote\system\Database\MySQLi\Connection.php:171
Stack trace:
#0 D:\xampp\htdocs\debitnote\system\Database\MySQLi\Connection.php(171): mysqli->real_connect('localhost', 'root', '', 'debitnote', 3306, '', 0)
#1 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#3 D:\xampp\htdocs\debitnote\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\debitnote\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\debitnote\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#6 D:\xampp\htdocs\debitnote\app\Controllers\Login.php(26): CodeIgniter\BaseModel->findAll()
#7 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(902): App\Controllers\Login->index()
#8 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 D:\xampp\htdocs\debitnote\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: MySQL server has gone away in D:\xampp\htdocs\debitnote\system\Database\MySQLi\Connection.php:215
Stack trace:
#0 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(382): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 D:\xampp\htdocs\debitnote\system\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
#2 D:\xampp\htdocs\debitnote\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 D:\xampp\htdocs\debitnote\system\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
#4 D:\xampp\htdocs\debitnote\system\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
#5 D:\xampp\htdocs\debitnote\app\Controllers\Login.php(26): CodeIgniter\BaseModel->findAll()
#6 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(902): App\Controllers\Login->index()
#7 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 D:\xampp\htdocs\debitnote\system\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 D:\xampp\htdocs\debitnote\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2025-01-02 09:47:32 --> Unable to connect to the database.
Main connection [MySQLi]: MySQL server has gone away
in SYSTEMPATH\Database\BaseConnection.php on line 418.
 1 SYSTEMPATH\Database\BaseConnection.php(588): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `userdata`
WHERE `username` = :username:', [...], false)
 3 SYSTEMPATH\Model.php(223): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(584): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Login.php(26): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Login->index()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 09:59:30 --> Invalid file: partials/approval_form.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('partials/approval_form.php')
 2 SYSTEMPATH\View\View.php(429): CodeIgniter\View\View->render('partials/approval_form', [...], true)
 3 APPPATH\Views\detail_engineer.php(61): CodeIgniter\View\View->include('partials/approval_form', [...])
 4 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\detail_engineer.php')
 5 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 6 SYSTEMPATH\Common.php(1142): CodeIgniter\View\View->render('detail_engineer', [], true)
 7 APPPATH\Controllers\Detail_engineer.php(40): view('detail_engineer', [...])
 8 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Detail_engineer->index()
 9 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Detail_engineer))
10 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 10:32:27 --> Too few arguments to function App\Models\EngineerModel::cancel(), 4 passed in D:\xampp\htdocs\debitnote\app\Controllers\Detail_engineer.php on line 154 and exactly 6 expected
in APPPATH\Models\EngineerModel.php on line 135.
 1 APPPATH\Controllers\Detail_engineer.php(154): App\Models\EngineerModel->cancel('0002-GSS-100724', 5, 'test', '1735788747_0d94897dd62a702c513a.jpg')
 2 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Detail_engineer->cancel()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Detail_engineer))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:40:51 --> Array to string conversion
in APPPATH\Controllers\Engineer_receipt.php on line 66.
 1 APPPATH\Controllers\Engineer_receipt.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Engineer_receipt.php', 66)
 2 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:41:01 --> Array to string conversion
in APPPATH\Controllers\Engineer_receipt.php on line 66.
 1 APPPATH\Controllers\Engineer_receipt.php(66): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Array to string conversion', 'APPPATH\\Controllers\\Engineer_receipt.php', 66)
 2 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 3 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 4 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:41:08 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 866.
 1 SYSTEMPATH\BaseModel.php(866): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(693): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\EngineerModel.php(177): CodeIgniter\Model->update()
 4 APPPATH\Controllers\Engineer_receipt.php(93): App\Models\EngineerModel->update_date_perdin('333', '2024-07-25', '2024-07-27')
 5 APPPATH\Controllers\Engineer_receipt.php(69): App\Controllers\Engineer_receipt->updateSOAndDatePerdin([...], [...], [...], [...], Object(App\Models\EngineerModel))
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:41:57 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 866.
 1 SYSTEMPATH\BaseModel.php(866): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(693): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\EngineerModel.php(177): CodeIgniter\Model->update()
 4 APPPATH\Controllers\Engineer_receipt.php(94): App\Models\EngineerModel->update_date_perdin('333', '2024-07-25', '2024-07-27')
 5 APPPATH\Controllers\Engineer_receipt.php(69): App\Controllers\Engineer_receipt->updateSOAndDatePerdin([...], [...], [...], [...], Object(App\Models\EngineerModel))
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:45:13 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 866.
 1 SYSTEMPATH\BaseModel.php(866): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(693): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\EngineerModel.php(177): CodeIgniter\Model->update()
 4 APPPATH\Controllers\Engineer_receipt.php(95): App\Models\EngineerModel->update_date_perdin('333', '2024-07-25', '2024-07-27')
 5 APPPATH\Controllers\Engineer_receipt.php(69): App\Controllers\Engineer_receipt->updateSOAndDatePerdin([...], [...], [...], [...], Object(App\Models\EngineerModel))
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:47:18 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 866.
 1 SYSTEMPATH\BaseModel.php(866): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(693): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\EngineerModel.php(177): CodeIgniter\Model->update()
 4 APPPATH\Controllers\Engineer_receipt.php(95): App\Models\EngineerModel->update_date_perdin('333', '2024-07-25', '2024-07-27')
 5 APPPATH\Controllers\Engineer_receipt.php(69): App\Controllers\Engineer_receipt->updateSOAndDatePerdin([...], [...], [...], [...], Object(App\Models\EngineerModel))
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2025-01-02 14:47:55 --> There is no data to update.
in SYSTEMPATH\BaseModel.php on line 866.
 1 SYSTEMPATH\BaseModel.php(866): CodeIgniter\Database\Exceptions\DataException::forEmptyDataset('update')
 2 SYSTEMPATH\Model.php(693): CodeIgniter\BaseModel->update(null, [])
 3 APPPATH\Models\EngineerModel.php(177): CodeIgniter\Model->update()
 4 APPPATH\Controllers\Engineer_receipt.php(95): App\Models\EngineerModel->update_date_perdin('333', '2024-07-25', '2024-07-27')
 5 APPPATH\Controllers\Engineer_receipt.php(69): App\Controllers\Engineer_receipt->updateSOAndDatePerdin([...], [...], [...], [...], Object(App\Models\EngineerModel))
 6 SYSTEMPATH\CodeIgniter.php(902): App\Controllers\Engineer_receipt->upload_receipts()
 7 SYSTEMPATH\CodeIgniter.php(457): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Engineer_receipt))
 8 SYSTEMPATH\CodeIgniter.php(340): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
